src/store/ChartStore.js

```diff
@@ -152,6 +152,56 @@ class ChartStore {
             yTolerance: 999999, // disable vertical scrolling
         });
 
+        // Monkey patch a custom positionMarkers function because the default selects only
+        // the static placementFunction from CrosshairStore
+        stxx.positionMarkers = function () {
```

Comment:
this should also be temporary hack. Refer to comment on making Crosshair not a `CIQ.Marker`.